frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -29,18 +29,19 @@ local Screen = Device.screen
 local MODE = {
     off = 0,
     page_progress = 1,
-    time = 2,
-    pages_left = 3,
-    battery = 4,
-    percentage = 5,
-    book_time_to_read = 6,
-    chapter_time_to_read = 7,
-    frontlight = 8,
-    mem_usage = 9,
-    wifi_status = 10,
-    book_title = 11,
-    book_chapter = 12,
-    bookmark_count = 13,
+    page_remaining = 2,
+    time = 3,
+    pages_left = 4,
+    battery = 5,
+    percentage = 6,
+    book_time_to_read = 7,
+    chapter_time_to_read = 8,
+    frontlight = 9,
+    mem_usage = 10,
+    wifi_status = 11,
+    book_title = 12,
+    book_chapter = 13,
+    bookmark_count = 14,
```

Comment:
Will changeing the mode valuse have an effect on existing users? 
Does it make sense to add `page_remaining` at the end as number 14?

Reply:
I honestly don't know - I thought they were like const values and changing them would just work.

Reply:
Thanks. I'd appreciate any thoughts on how to make it clearer. I'm seeing it as something you put up instead of the "Current page" (and vice versa) one since having both seems redundant.

Reply:
Please be sure of that :) 
I think the items we added the most recently were added last.
But looking at the code, it feels it would be fine with stuff added at any place in the list (it's possible we made it better because it was a mess previously and these kind of stuff were risky :)
Please just test that it's fine with a user previous custom ordering. (Can't test myself atm.)

There's also another ordered list as a comment at top of ReaderFooter:applyFooterMode(mode). Dunno if it should be updated.

Reply:
Thanks very much!

I have tried turning on/off all the status bar options and moving them about and haven't seen any issues. Is that sufficient, or should I try something else? 

The comment you mentioned reads

```
    -- three modes switcher for reader footer
    -- 0 for footer off
    -- 1 for footer page info
    -- 2 for footer time info
    -- 3 for footer next_chapter info
    -- 4 for battery status
    -- 5 for progress percentage
    -- 6 for from statistics book time to read
    -- 7 for from statistics chapter time to read
    -- 8 for front light level
    -- 9 for memory usage
    -- 10 for Wi-Fi status
    -- 11 for book title
    -- 12 for current chapter
    -- 13 for bookmark count
```

and the "local mode" reads

```
    off = 0,
    page_progress = 1,
    page_remaining = 2,
    time = 3,
    pages_left = 4,
    battery = 5,
    percentage = 6,
    book_time_to_read = 7,
    chapter_time_to_read = 8,
    frontlight = 9,
    mem_usage = 10,
    wifi_status = 11,
    book_title = 12,
    book_chapter = 13,
    bookmark_count = 14,
```

Should they line up? I'm not sure how to handle the  "footer page info" and "footer time info" entries.

Reply:
> The comment you mentioned reads
    -- three modes switcher for reader footer
    -- 0 for footer off
    -- 1 for footer page info

Yep, and I don't even understand what this comment is trying to say :) It has been completed over the years with the added items - so, I guess they should line up for consistency.
But I'd also be happy if you just deleted the whole comment - so there's no second set to keep updated for nothing.

> should I try something else?

I just checked:
- no ordering done before applying this PR - then PR applied - all ok.
- previous user ordering done before applying this PR - then PR applied - ordering kept, the new item is put at the end of the user order

So, I guess it's ok.